internal/agent/reset.go

```diff
@@ -14,27 +13,32 @@ import (
 	hook "github.com/kairos-io/kairos/v2/internal/agent/hooks"
 	"github.com/kairos-io/kairos/v2/internal/bus"
 	"github.com/kairos-io/kairos/v2/internal/cmd"
+	"github.com/kairos-io/kairos/v2/pkg/action"
 	"github.com/kairos-io/kairos/v2/pkg/config"
 	"github.com/kairos-io/kairos/v2/pkg/config/collector"
+	"github.com/kairos-io/kairos/v2/pkg/elementalConfig"
 
 	"github.com/mudler/go-pluggable"
 	"github.com/pterm/pterm"
 )
 
 func Reset(dir ...string) error {
+	// TODO: Enable args? No args for now so no possibility of reset persistent or overriding the source for the reset
+	// Nor the auto-reboot via cmd?
```

Comment:
afaict `--reset-persistent` flag was set through `r.Data` in this code. Why can't we keep it this way?

Reply:
This comment relates for the cli being called. If you call kairos-agent reset  there is no flags that can be passed manually to override the default config. Im just thinking that maybe we should be able to override the config via flags.